boards/shields/nrf7002ek/nrf7002ek_coex.overlay

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	nrf_radio_coex: nrf7002-coex {
+		status = "okay";
+		compatible = "nordic,nrf700x-coex";
```

Comment:
I can't find the bindings file for this compatible?

Reply:
Good catch, the file was in a different directory downstream and missed adding it, will fix this. Thanks.